.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

It is not necessary to add the build directory to your .gitignore (which wouldn't work if you use `meson setup builddir/` or even "builddir-release" alongside "builddir-debug").

Reply:
LGTM.